### DISCUSSION TITLE
NPC VS NPC Whitelist and blacklist

### DISCUSSION THREAD

**Discussion by EspectroW94** [AscensionGameDev/Intersect-Engine#2261] [Ideas]
Could be implement a "white list" and a "black list" where, if the "white list" is checked, it attacks all the other NPCs that are not there. And if the "blacklist" is selected, will it only attack the NPCs that are there? This would be very practical when you have many npc

I mean this:
![image](https://github.com/AscensionGameDev/Intersect-Engine/assets/127634629/28cef85a-696c-4f31-b2db-00ef81f3bcf3)

**Comment by WeylonSantana** [comment]
this is amazing idea

**Comment by PyroTech03** [comment]
Please for the love of God....

**Comment by MGMarauder** [comment]
This would be great, not having this is the main reason I don't have Combative friendly NPCs.

**Comment by dhmello** [comment]
i agree :)
